modules/gdscript/gdscript_tokenizer.h

```diff
@@ -142,6 +142,7 @@ class GDScriptTokenizer {
 		TK_ERROR,
 		TK_EOF,
 		TK_CURSOR, //used for code completion
+		TK_TOOLTIP,
```

Comment:
creating a new token would break the parser unless every corner cases were handled.
ex: if `TK_TOOLTIP` inside a function the parser complains `Expected end of statement after expression, got ...` inside `_parse_block()`
and we check the next and previous tokens for parsing so we need to tell the parser that if it's a tool tip token just skip it.